### PR TITLE
CV2-2256: Add more options for search language filter 

### PR DESCRIPTION
**Pull request by melsawy**
## Description

Add more options for search language filter so user can filter by `Conversation Language`, `Media language`, and `Report Language`

References: CV2-2256

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

1. Go to list items.
2. Click `Add filter` and choose language
3. user can filter by Media language, Report language and Conversation language

## Things to pay attention to during code review

Please describe parts of the change that require extra attention during code review, for example:

- File FFFF, line LL: This refactoring does this and this. Is it consistent with how it’s implemented elsewhere?
- Etc.

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] I have removed the /* eslint-disable @calm/react-intl/missing-attribute */ from any files I've worked on and added a `description` prop to all `<FormattedMessage />` components that were missing it.
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

**Review by caiosba — COMMENTED** [review]
@melsawy , I would like to try this code locally... can I use the `develop` branch of Check API or I need to use a specific branch of Check API?

**Comment by melsawy** [comment]
@caiosba  you can use the develop branch of Check API and you should run the following rake tasks to index existing data

1. bundle exec rails check:migrate:index_report_language
2. bundle exec rails check:project_media:sync_es_nested_field['field:requests']

File: src/app/components/search/LanguageFilter.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
+/* eslint-disable @calm/react-intl/missing-attribute */
```
Comment (amoedoamorim) [inline src/app/components/search/LanguageFilter.js:1]: Please remove this eslint-disable directive

**Review by amoedoamorim — APPROVED** [review]
Working as expected. Let's try to use the css modules for styling next time!

**Review by caiosba — COMMENTED** [review]
Looks good and work as expected locally. I agree with @amoedoamorim regarding the `Eslint` directive.

File: src/app/components/search/LanguageFilter.js (hunk 1 above)
Comment (melsawy) [reply src/app/components/search/LanguageFilter.js]: Done

**Review by caiosba — APPROVED** [review]

**Pull request merged**